.github/workflows/ci.yml

```diff
@@ -172,13 +175,14 @@ jobs:
       run: |
         cd $HOME/libzim/build
         ninja download_test_data
-        meson test --verbose
+        meson test -t 0 --verbose
```

Comment:
I guess that this change was needed to find out the value for the waiting factors on some CI platforms, but it looks like the new unit test simply gets stuck on some of them. In any case this change must not make its way into master.

Reply:
Yes, this is removed.